server/src/models/Review.js

```diff
@@ -27,7 +27,7 @@ class Review extends Model {
           to: "holidays.id",
         },
       },
-      users: {
+      user: {
```

Comment:
Nice fix! That probably means that the `holidays` relation should also be singular